buildSrc/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 
 repositories {
     mavenCentral()
-    jcenter()
+    gradlePluginPortal()
 }
```

Comment:
This can probably removed altogether

Reply:
gradlePluginPortal() has to stay because otherwise we get unresolved dependencies